pkgs/top-level/python-packages.nix

```diff
@@ -4213,7 +4213,7 @@ in {
 
   mecab-python3 = callPackage ../development/python-modules/mecab-python3 { };
 
-  MechanicalSoup = callPackage ../development/python-modules/MechanicalSoup { };
+  mechanicalsoup = callPackage ../development/python-modules/mechanicalsoup { };
```

Comment:
Please add an alias for this to python-aliases.nix.

Reply:
Done, thanks.